ocw/lib/ec2.py

```diff
@@ -325,7 +325,12 @@ def vpc_can_be_deleted(self, resource_vpc, vpc_id) -> bool:
 
     def report_cleanup_results(self, vpc_errors: list, vpc_notify: list, vpc_locked: list) -> None:
         if len(vpc_errors) > 0:
```

Comment:
This `if` is redundant with the block below.

Reply:
agree with you but unfortunately  I pressed merge button just before your comment popup :disappointed: